src/zorak/cogs/admin/verification_on_verified.py

```diff
@@ -19,9 +22,43 @@ def __init__(self, bot):
     def is_verified(self, member):
         if "✅" in [role.name for role in member.roles]:
             return True
+        
+    async def send_wrong_button_message(self, guild, member):
+        button_message = f"""
+            Hi there, {member.mention}
+            you have pressed the wrong button to verify yourself. 
+
+            Make sure you press the **GREEN** button to verify yourself.
+            Please join the server again and try again.
+            """
+        # Send Welcome Message
+        try:
+            await member.send(button_message)
+        except discord.errors.Forbidden as catch_dat_forbidden:
+            logger.debug(f'{member.name} cannot be sent a DM.')
+
+    @discord.ui.button(label="Verify!", row=0, style=discord.ButtonStyle.red)
+    async def verify_button_callback_red_1(self, button: discord.ui.Button, interaction: discord.Interaction):
+        """
+        This is a dummy button, if pressed kicks user.
+        """
+        user = interaction.user
+        await self.send_wrong_button_message(interaction.guild, user)
+        await user.kick(reason="User failed to verify.")
```

Comment:
We should be more explicit in the logging here. Failed to verify because they clicked a red button.